### DISCUSSION TITLE
GTA V & Hitman

### DISCUSSION THREAD

**Discussion by ghost** [legendary-gl/legendary#195] [General]
Can I run GTA V and Hitman which is in my epic games library

**Comment by derrod** [comment]
Yes.

**Reply by ghost** [reply]
Is there any way to pause or resume game download, because while downloading large games there might be network outage. And when we do legendary install gamename, it will download it from start.

**Reply by CommandMC** [reply]
You can pause a download by pressing CTRL + C when it's downloading. Starting the download again will then only download the files that still need to be downloaded. It will still start at 0%, but the download size will be less and you *should* as far as I remember get a message about the download being resumed

**Reply by ghost** [reply]
That's Great thanks for the support.

**Comment by ghost** [comment]
I am getting  error as attached in the screenshot after installing GTA V from my epic library. Please help me with this. I have already installed rockstar launcher through lutris.
![Screenshot_20210110_144858](https://user-images.githubusercontent.com/6077624/104119175-cfe1ba00-5353-11eb-80c8-bc26c6164890.png)

**Comment by ghost** [comment]
1. the error messsage window is hiding any thing that might help the developer troubleshoot things with you. It would have been helpful if you moved it to the side so we could see the terminal output.
2. this sounds more like a game not being compatible with Wine/Linux than it is a Legendary issue. But I'm not the developer
3. It's probably because GTA5 is a Windows game and not a Mac/Linux game and Wine ran into a problem. Better to do some googeling or try to get support from the proper place - WINE/Linux gaming support channels

**Reply by ghost** [reply]
Okay, I will try that.

**Comment by CommandMC** [comment]
GTA runs perfectly using Proton. See the wiki on how to use it with Legendary
And yes, that is a Wine issue

**Reply by ghost** [reply]
I got that  it is not a legendary issue, it is related to wine. Thanks for the help
